core/primitives/big_int.hpp

```diff
@@ -21,6 +21,38 @@ namespace fc::primitives {
     using cpp_int::cpp_int;
   };
 
+  static inline const BigInt operator+(const BigInt &lhs, const BigInt &rhs) {
```

Comment:
`readability-const-return-type` - cpp-lint advices removing unnecessary const in return.

Reply:
Fixed